test/test_integration_check.py

```diff
@@ -0,0 +1,192 @@
+    def test_check_without_id_returns_help(self):
+        with redirect_stderr(self.capture_stream):
+            self.invoke_cli(self.cli_auth_params,
+                            ['check'], exit_code=1)
+        self.assertIn("Error the following arguments are required:", self.capture_stream.getvalue())
+
+    @integration_test(True)
+    def test_check_short_with_help_returns_check_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-h'])
+        self.assertIn("Name:\n  check", output)
+
+    @integration_test(True)
+    def test_check_long_with_help_returns_check_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '--help'])
+        self.assertIn("Name:\n  check", output)
+
+    '''
+    Tests to check for a privilege on a resource with a default role
+    '''
+    @integration_test(True)
+    def test_check_existent_privilege_on_permitted_resource_with_default_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-i', 'dev:variable:somevariable', '-p', 'read'])
+        self.assertIn(output, "true\n")
+
+    @integration_test(True)
+    def test_check_non_existent_privilege_on_permitted_resource_with_default_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-i', 'dev:variable:somevariable', '-p', 'execute'])
+        self.assertIn(output, ["true\n", "false\n"])
+
+    @integration_test(True)
+    def test_check_existent_privilege_on_non_permitted_resource_with_default_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-i', 'dev:webservice:someservice', '-p', 'read'])
+        self.assertIn(output, "false\n")
+
+    @integration_test(True)
+    def test_check_non_existent_privilege_on_non_permitted_resource_with_default_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-i', 'dev:policy:somepolicy', '-p', 'execute'])
+        self.assertIn(output, ["true\n", "false\n"])
+
+    '''
+    Tests to check for a privilege on a resource with a specified role
+    '''
+    @integration_test(True)
+    def test_check_existent_privilege_on_permitted_resource_with_permitted_specified_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-i', 'dev:variable:somevariable', '-p', 'read', '-r', 'dev:user:permitted_user'])
+        self.assertIn(output, "true\n")
+
+    @integration_test(True)
+    def test_check_non_existent_privilege_on_permitted_resource_with_permitted_specified_role(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['check', '-i', 'dev:variable:somevariable', '-p', 'execute', '-r', 'dev:host:permitted_host'])
+        self.assertIn(output, "false\n")
```

Comment:
You have a handful of tests here where you are supplying a `permitted_*` role ID but expecting a false result. I think it would be a bit easier to follow if you either updated your policy/tests so that these return true, or use a different role which is clearly intended not to have the privilege.